packages/expo/cli/start/platforms/AppIdResolver.ts

```diff
@@ -0,0 +1,63 @@
+import { getConfig, getProjectConfigDescriptionWithPaths } from '@expo/config';
+
+import { CommandError, UnimplementedError } from '../../utils/errors';
+import { get } from '../../utils/obj';
+
+export class AppIdResolver {
```

Comment:
Add a docblock explaining this is for application identifiers like the Android application ID and Apple bundle identifier (as opposed to the EAS project ID)